src/sidebar-overlay/utils.ts

```diff
@@ -70,3 +71,37 @@ export const setSidebarState = async (enabled: boolean) =>
 
 export const getExtUrl = (location: string) =>
     browser.runtime ? browser.runtime.getURL(location) : location
+
+// TODO: Perhaps move RPC calls to some sort of a manager.
```

Comment:
This is similar to our idea with overhauling our RPC system :)
https://www.notion.so/worldbrain/55431a6dae7043b79d04e4633c01f798?v=06e036ccc50b492fab084cdd30a979eb&p=fa36567389f84513a43934aa908d80ec